pkg/apis/provisioning/v1alpha5/labels.go

```diff
@@ -30,10 +30,11 @@ var (
 	// Karpenter specific domains and labels
 	KarpenterLabelDomain = "karpenter.sh"
 
-	ProvisionerNameLabelKey         = Group + "/provisioner-name"
-	DoNotEvictPodAnnotationKey      = Group + "/do-not-evict"
-	EmptinessTimestampAnnotationKey = Group + "/emptiness-timestamp"
-	TerminationFinalizer            = Group + "/termination"
+	ProvisionerNameLabelKey           = Group + "/provisioner-name"
+	DoNotEvictPodAnnotationKey        = Group + "/do-not-evict"
+	DoNotConsolidateNodeAnnotationKey = KarpenterLabelDomain + "/do-not-consolidate"
```

Comment:
I'm not sure if this is a good idea yet, but just a thought. If we make this a label, there are a couple of interactions here that are *very* interesting. 

A pod can request a node that doesn't consolidate by node selector. You can enable/disable consolidation at the provisioner level via standard requirements.

Reply:
I prefer an explicit enabling/disabling of consolidation rather than having a magic label to turn it on/off.  That's also large departure from the design document. Someone can experiment with other options in a follow-on PR.